go/cmd/tz-batch-create-job-definition/batch-create-job-definition.go

```diff
@@ -90,7 +90,11 @@ func main() {
 				ReadonlyRootFilesystem: aws.Bool(true),
 			},
 			RetryStrategy: &batch.RetryStrategy{
-				Attempts: aws.Int64(int64(jobDef.RetryAttempts)),
+				Attempts:       aws.Int64(int64(jobDef.RetryAttempts)),
+				EvaluateOnExit: []*batch.EvaluateOnExit{{
+						Action:     aws.String(batch.RetryActionExit),
+						OnExitCode: aws.String("137"),
```

Comment:
do we have a link to the exit code 137 saying that it is OOM only or some other memory issue may also have same exit code 137?

Reply:
https://tldp.org/LDP/abs/html/exitcodes.html

Reply:
updated to use reason pattern `".*OutOfMemoryError.*"`